backend/migrator/migration/prod/2.14/0001##worksheet.sql

```diff
@@ -0,0 +1,24 @@
+CREATE TABLE worksheet (
```

Comment:
Maybe we can put all the DDL, DMLs into the same sql file so that we use the same transaction to apply changes. All succeed or fail, instead of failing in the middle.

Reply:
Updated